examples/nextjs-tiptap-novel/src/app/api/upload/_route.ts

```diff
@@ -0,0 +1,26 @@
+// import { put } from "@vercel/blob";
+// import { NextResponse } from "next/server";
```

Comment:
Should the upload feature also be disabled since there's no route to handle it?

Reply:
Yes!